### PR TITLE
[NFV] TRex package installation

### PR DESCRIPTION
**Pull request by jlausuch**
Instead of building T-Rex, use the existing tarball where everything is pre-built. 

- Related ticket: https://progress.opensuse.org/issues/34891
- Verification run: http://fromm.arch.suse.de/tests/294

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi @schlad  can you please review this?

**Comment by schlad** [comment]
LGTM

File: tests/nfv/trex_installation.pm [hunk 1]
```diff
@@ -23,24 +19,20 @@ use mmapi;
...
-    # Compile Trex libraries
-    assert_script_run("cd $trex_dest/linux_dpdk");
-    assert_script_run("./b configure", 300);
-    assert_script_run("./b build",     1400);
+    assert_script_run("cp $trex_dest/cfg/simple_cfg.yaml /etc/trex_cfg.yaml");
 }
 
 1;
-
+# vim: set sw=4 et:
```
Comment (rwx788) [inline tests/nfv/trex_installation.pm]: @asmorodskyi has removed these setting from most of the files, I guess now we should avoid it.

File: tests/nfv/trex_installation.pm [hunk 2]
```diff
@@ -23,24 +19,20 @@ use mmapi;
 use serial_terminal 'select_virtio_console';
 
 sub run {
-    my $trex_repo = "https://github.com/cisco-system-traffic-generator/trex-core.git";
-    my $trex_dest = "/tmp/trex-core";
+    my $trex_version = get_var('TG_VERSION');
```
Comment (rwx788) [inline tests/nfv/trex_installation.pm]: What's about using get_required_var here to fail for a clear reason when variable is not set? WDYT?

Comment (jlausuch) [reply tests/nfv/trex_installation.pm]: it makes sense as it is required. It's updated now. Thanks.

File: tests/nfv/trex_installation.pm (hunk 1 above)
Comment (jlausuch) [reply tests/nfv/trex_installation.pm]: Removed :)

**Pull request merged**